README.md

```diff
@@ -51,7 +51,17 @@ which generates a single file `output/main` which can be run with `scheme` or `p
 scheme --program output/main
 ```
 
-## Vendored Dependencies
+## Development
+
+### Running the tests
+
+To run all tests, run `npm run test`. This will run both the scheme runtime unit tests and the snapshot tests.
+
+To overwrite old snapshots with the latest output, run `npm run test -- -a "--accept"`.
+To add a new snapshot, create a file called `Snapshot.X.purs` where `X` indicates what is being tested.
+If a snapshot needs a dependency, install it by running `spago -x test-snapshots/spago.dhall install <packages...>`. The snapshots have their own `spago.dhall` file so as not to pollute the `purescm` binary with unneeded dependencies.
```

Comment:
```suggestion
If a snapshot needs a dependency, install it by running `cd test-snapshots && spago install <packages...>`.
```

Reply:
Cool, I meant to ask you about whether we should update this.